packages/serp-report/manifest-v2.json

```diff
@@ -199,21 +199,15 @@
       ],
       "run_at": "document_end",
       "css": [
-        "/vendor/@whotracksme/serp-report/src/content_scripts/index.css"
+        "vendor/@whotracksme/serp-report/src/content_scripts/serp-report.css"
       ],
       "js": [
-        "/vendor/@whotracksme/tracker-wheel/src/index.js",
-        "/vendor/@whotracksme/serp-report/src/content_scripts/index.js"
+        "vendor/@whotracksme/serp-report/src/content_scripts/serp-report.js"
       ]
     }
   ],
-  "background": {
-    "scripts": [
-      "/vendor/@whotracksme/serp-report/background/data.js",
-      "/vendor/@whotracksme/serp-report/background/index.js"
-    ]
-  },
   "web_accessible_resources": [
-    "vendor/@whotracksme/serp-report/assets/iframe/index.html"
+    "vendor/@whotracksme/serp-report/src/pages/iframe/index.html",
+    "vendor/@whotracksme/ui/src/tracker-wheel.js"
```

Comment:
It is a dependency for the `serp-report`, so I think it should be here for now (it may be moved out from the `/ui` in next steps after all).

Reply:
It mast be, as it is used by the `import()` in content script. If it is not accessible, it can not be imported in that way. It is added to the `dnr-extension` as well, but in the corresponding PR.

Reply:
Ok, make sense. Wonder what reviewers will say about it :)